packages/react/src/components/Button/Button.test.tsx

```diff
@@ -13,7 +15,7 @@ describe('Button component', () => {
     const onClickMock = jest.fn()
     const { component: { getByTestId } } = renderWithTheme(<Button label="button" onClick={onClickMock} testID="btn-test" />)
 
-    fireEvent(getByTestId('btn-test'), new MouseEvent('click', { bubbles: true }))
+    fireEvent.click(getByTestId('btn-test'))
```

Comment:
acho que o test `should render correctly when disabled is true`
poderia ser `should render correctly when the variant is contained and disabled is true` pra facilitar a leitura.
Primeiramente eu pensei que estava faltando o teste do contained, depois me dei conta que ele é o default